elytron-oidc-client/subsystem/src/test/java/org/wildfly/extension/elytron/oidc/ExpressionsTestCase.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2021 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.extension.elytron.oidc;
+
+import static org.wildfly.extension.elytron.oidc.ElytronOidcSubsystemDefinition.ELYTRON_CAPABILITY_NAME;
+
+import org.jboss.as.controller.RunningMode;
+import org.jboss.as.controller.capability.registry.RuntimeCapabilityRegistry;
+import org.jboss.as.controller.extension.ExtensionRegistry;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+import org.jboss.as.controller.registry.Resource;
+import org.jboss.as.subsystem.test.AbstractSubsystemTest;
+import org.jboss.as.subsystem.test.AdditionalInitialization;
+import org.jboss.as.subsystem.test.KernelServices;
+import org.junit.Assert;
+//import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Subsystem parsing test case.
+ *
+ * <a href="mailto:araskar@redhat.com">Ashpan Raskar</a>
+ */
+public class ExpressionsTestCase extends AbstractSubsystemTest {
+
+    private OidcConfigService configService;
+    private KernelServices services = null;
+
+    public ExpressionsTestCase() {
+        super(ElytronOidcExtension.SUBSYSTEM_NAME, new ElytronOidcExtension());
+    }
+
+    @Test
+    public void testExpressions() throws Throwable {
+        if (services != null) return;
+        String subsystemXml = "oidc-expressions.xml";
+        services = super.createKernelServicesBuilder(new DefaultInitializer()).setSubsystemXmlResource(subsystemXml).build();
+        if (! services.isSuccessfulBoot()) {
+            Assert.fail(services.getBootError().toString());
+        }
+        configService = OidcConfigService.getInstance();
```

Comment:
I think the configService can be removed since it's not used in this test.

Reply:
Removed